raglweb/__init__.py

```diff
@@ -284,11 +284,41 @@ def player(profile_id):
 def info():
     return render_template('info.html', start_time=_cfg['start_time'])
 
+def _make_division_prefix(division_title):
```

Comment:
We now have:
```
_make_division_prefix('Masters')
Out[46]: 'RAGL-S10-MASTER-'

_make_division_prefix('Minions Alpha')
Out[47]: 'RAGL-S10-MINIONA-'

_make_division_prefix('Minions Bravo')
Out[48]: 'RAGL-S10-MINIONB-'
```

Reply:
Looks good to me: can you squash the commits together and push force?